.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "geoblacklight"]
 	path = geoblacklight
 	url = https://github.com/CottageLabs/geoblacklight.git
+
+[submodule "willow_sword"]
+	path = willow_sword
+	url = git@github.com:CottageLabs/willow_sword.git
```

Comment:
We can set up the demo server for ssh access to github though. It just needs public access, and also our production environments shouldn't have this problem. Any *build* environment where the repo needs to be pulled will have it.